client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -74,10 +74,24 @@ class VerificationCodeForm extends Component {
 		const isError = !! twoFactorAuthRequestError;
 
 		let helpText = translate( 'Type in the code generated by your Authenticator mobile application.' );
+		let labelText = translate( 'Verification Code' );
+		let smallPrint;
 		if ( twoFactorAuthType === 'sms' ) {
 			helpText = translate( 'A text message with the verification code was just sent to your phone number.' );
 		}
 
+		if ( twoFactorAuthType === 'backup' ) {
+			helpText = translate( "If you can't access your phone enter one of the 10 backup codes that were provided " +
+				'when you set up two-step authentication to continue.' );
+			labelText = translate( 'Backup code' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Backup Codes' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).